pkg/cidata/template.go

```diff
@@ -105,6 +105,7 @@ type TemplateArgs struct {
 	VirtioPort                      string
 	Plain                           bool
 	TimeZone                        string
+	LimaInit                        bool
```

Comment:
this might be more clear 
```suggestion
	NoCloudInit                        bool
```